xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1443,6 +1443,10 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
       {
         stream = new CDemuxStream();
         stream->type = STREAM_DATA;
+        // Video (mp4) from GoPro cameras can had a 'meta' track used for a file repair containing
+        //  'fdsc' data, this is also called the SOS track. Ignore it or we will glitch bad on startup.
+        if (pStream->codec->codec_tag == MKTAG('f','d','s','c'))
```

Comment:
adding a discareded stream is not really what you want to achieve, right? better not adding it to the map at all

Reply:
Can you explain exactly what is desired here?
Returning nullptr from this function causes playback to fail 
```
17:03:30.360 T:1700598560   ERROR: CDVDDemuxFFmpeg::AddStream - internal error, stream is null
```

Reply:
do not allocate a packet and return empty (bReturnEmpty) in this case